views/default/css/admin.php

```diff
@@ -816,16 +816,16 @@
 	z-index:1;
 }
 .friends-picker-navigation-l {
-	background: url("<?php echo elgg_get_site_url(); ?>_graphics/friendspicker.png") no-repeat left top;
+	background: url("<?= elgg_get_simplecache_url("friendspicker.png"); ?>") no-repeat left top;
```

Comment:
We don't allow `<?=` currently. I think we should, as it's always enabled now, and useful.